controllers/medusa/cassandrabackup_controller.go

```diff
@@ -215,6 +217,6 @@ func backupFinished(backup *medusaapi.CassandraBackup) bool {
 
 func (r *CassandraBackupReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&medusaapi.CassandraBackup{}).
+		For(&medusaapi.CassandraBackup{}, builder.WithPredicates(predicate.GenerationChangedPredicate{})).
```

Comment:
Does that prevent from reconciling on status changes?

Reply:
Yep, it filters all the reconcile requests that did not change the real object (as the generation stays the same if we update the status).